osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/SecurityKeyProvider.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2021 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.factories;
+
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.SecurityKeyType;
+
+/**
+ * Functional interface used in connectors to obtain keys.
+ * @see Hls5Connector
```

Comment:
The other SecureDlmsConnector(s) (the Lls1Connector comes to mind) may be offended ;). (You could consider referring to "classes extending `SecureDlmsConnector`" instead.)

Reply:
You're right, I didn't mean to discriminate against other connectors :-), so I generalised it to SecureDlmsConnector.